src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ClientCertificates.cs

```diff
@@ -201,7 +201,6 @@ public void Manual_SendClientCertificateWithNoEKUToRemoteServer_OK()
         }
 
         [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "dotnet/corefx #20010")]
-        [ActiveIssue(9543)] // consistently fails with errors about client credentials being invalid
```

Comment:
Does this actually fix #9543?  I'd have expected this to address the SocketsHttpHandler sporadic failures but not the WinHttpHandler deterministic failure that happens even when running this test on its own.

Reply:
That issue has gotten rather confusing but the original root cause was about parallel PFX import of the client certificates.  That problem would happen with any Windows handler (SocketsHttpHandler or WinHttpHandler).  The issue is due to how SCHANNEL/CRYPTO places temporary private key files on disk.

I'd like to close #9543 and then if new problems come up, we can open a fresh issue.

Reply:
Interesting.  Now I'm getting an error.  I opened #29725 for the WinHttpHandler issue.

The error is very specific for WinHttpHandler.  It is getting ERROR_WINHTTP_CLIENT_CERT_NO_ACCESS_PRIVATE_KEY at times if all the tests run in parallel. This is a different kind of error even though it is using the updated PFX files.

I think there is something else going on with PFX import beyond getting rid of the friendlyname in the private key PFX file to solve the temporary filename on disk.

Reply:
Decided to switch to X509KeyStorageFlags.PersistKeySet for now.  Will track work to switch to EphemeralKeySet via #29728.